### PR TITLE
Tailwind Rollback

### PR DESCRIPTION
**Pull request by JackB47**
This PR contains the rollback for Tailwind to v1.9.6 as this is the latest version to support IE11. The branch contains style corrections to make the site compatible with IE11 as well as handling instances where newer tailwind classes aren't supported.

There are also a few adjustments to account for accessibility based on what happens on https://service-manual.nhs.uk/design-system

### CODE REVIEW

File: src/_includes/components/card.njk [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
-      <div class="p-6">
-        <h2 class="text-blue group-hover:text-dark-pink underline group-hover:no-underline text-xl lg:text-2xl font-semibold">{{card.label}}</h2>
-        {% if card.excerpt %}
-          <p class="text-primary mt-4 mb-3 lg:text-lg line-clamp-2">{{card.excerpt}}</p>
-        {% endif %}
-        
-      </div>
-    </a>
\ No newline at end of file
+    </div>
```
Comment (JamieBradders) [inline src/_includes/components/card.njk:11]: Hey @JackB47 are we missing a closing tag for the `div` in the `{% if %}` block?

File: src/_includes/layouts/base.njk [hunk 2]
```diff
@@ -12,7 +12,8 @@
     {% seo "" %}
 </head>
 
-<body class="{{bodyClass}} h-screen flex flex-col" >
+{# @TODO - add h-screen and override if on ie11 #}
```
Comment (JamieBradders) [inline src/_includes/layouts/base.njk:15]: Is this done now @JackB47?

File: tailwind.config.js [hunk 3]
```diff
@@ -1,52 +1,61 @@
...
-    
+    content: ["./dist/**/*.html"],
+
     // Manually whitelist style classes for PurgeCSS to ignore
     options: {
       whitelist: [],
-    }
-},
+    },
+  },
```
Comment (JamieBradders) [inline tailwind.config.js:9]: Does purge still work @JackB47 or do we need to incorporate this manually?

**Review by JamieBradders — CHANGES_REQUESTED** [review]
Couple of quick pointers to look at @JackB47 but we're onto a winner otherwise 👍

File: src/_includes/components/card.njk (hunk 1 above)
Comment (JackB47) [reply src/_includes/components/card.njk:11]: Nope there's just some awkward formatting going on - i've resaved without it for clarity 👍🏼

File: tailwind.config.js (hunk 3 above)
Comment (JackB47) [reply tailwind.config.js:9]: I believe it does yeah, the dist file hasn't churned every single tw class out

**Pull request closed without merging**